playbooks/roles/ocp-fio/defaults/main.yml

```diff
@@ -0,0 +1,20 @@
+---
+
+fio_install_operator: true
+fio_catalogsource_image: ""            # brew.registry.redhat.io/rh-osbs/iib:276833
+fio_catalogsource: "redhat-operators"
```

Comment:
This is set in the task. Please verify once.

Reply:
yes when only `fio_index_image` is defined, then it creates catalog source with name `file-integrity-operator` and when  `fio_e2e` is set then it must use catalog source name `qe-app-registry`. 

If `fio_index_image` is empty, it will use default `redhat-operators`

I will use one catalog source name `qe-app-registry` if `fio_index_image` is set.